files/image_config/kubernetes/scripts/kube_join.sh

```diff
@@ -0,0 +1,235 @@
+#! /bin/bash
+
+# Redirect any stdout & stderr into logger
```

Comment:
i really think we should do this in python. using some kubernates python client like https://github.com/kubernetes-client/python . down the road, we will have to integrate kuber control with service restart/start feature, and it is better to unified to the same framework to do it.

Reply:
Agree. Per offline discussion, upon proper brainstorming, we would come up with an appropriate infrastructure and then we could fit this script better in that. For now, we will proceed with this bash script.